src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectsTableRow/RedirectsTableRow.js

```diff
@@ -1,21 +1,41 @@
-import cx from "classnames";
+import { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { fetchContentItem } from "shell/store/content";
+import cx from "classnames";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import {
   faArrowRight,
   faExternalLinkAlt,
   faFile,
   faFileAlt,
+  faLink,
   faTrashAlt,
 } from "@fortawesome/free-solid-svg-icons";
 
 import { Button } from "@zesty-io/core/Button";
+import { Url } from "@zesty-io/core/Url";
 
 import styles from "./RedirectsTableRow.less";
 export default function RedirectsTableRow(props) {
+  const dispatch = useDispatch();
+  const [pathName, setPathName] = useState("");
+  const [pathLink, setPathLink] = useState("");
+
+  useEffect(() => {
+    if (props.targetType === "page") {
+      dispatch(fetchContentItem(props.target)).then((res) => {
+        if (res && res.data) {
+          setPathLink(res.data[0].meta.contentModelZUID);
```

Comment:
With naming we should prefer to keep the name and value conceptual the same. e.g. `setModelZuid` and `setPath`